go.mod

```diff
@@ -30,9 +30,9 @@ require (
 	github.com/pavel-v-chernykh/keystore-go v2.1.0+incompatible
 	github.com/prometheus/common v0.9.1
 	go.opencensus.io v0.22.0 // indirect
-	k8s.io/api v0.17.3
-	k8s.io/apimachinery v0.17.3
-	k8s.io/client-go v0.17.3
+	k8s.io/api v0.17.4
```

Comment:
You may want to upgrade directly to 0.17.5 as it fixed an issue in client-go that can impact the operator
https://github.com/kubernetes/kubernetes/pull/89976